src/test/java/com/google/javascript/cl2dts/interface_usage.ts

```diff
@@ -2,4 +2,6 @@
 import I from 'goog:interface_exp';
 class C implements I {
     method(): number { return 0; }
-}
\ No newline at end of file
+}
+
+var a: number = I.staticMethod();
```

Comment:
can you also add a test case with an interface that has no static members? I think that is missing